lib/manager/docker-compose/extract.spec.ts

```diff
@@ -11,6 +11,11 @@ const yamlFile3 = readFileSync(
   'utf8'
 );
 
+const yamlFile3NoVersion = readFileSync(
+  'lib/manager/docker-compose/__fixtures__/docker-compose.3-no-version.yml',
+  'utf8'
+);
+
 describe('lib/manager/docker-compose/extract', () => {
```

Comment:
```suggestion
describe(getName(__filename), () => {
```

import from `test/util` 😏

Reply:
oh, fancy! Thank you!